.travis.yml

```diff
@@ -16,7 +16,7 @@ android:
     - platform-tools
     - tools
     - build-tools-27.0.3
-    - android-25
+    - android-27
     - android-21
```

Comment:
why is this line still there?

Reply:
Missed to remove android-21, removed in latest commit

Reply:
Actually, android-21 is for the emulator config. Added it back with a comment